static/js/data_handler.js

```diff
@@ -74,6 +75,46 @@ export let dataHandler = {
     },
     createNewCard: function (cardTitle, boardId, statusId, callback) {
         // creates new card, saves it and calls the callback function with its data
-    }
+    },
     // here comes more features
+    submitNewTitle: function submit_new_title(event) {
```

Comment:
Here what you do is:
- define a function named `submit_new_title`
- assign this function to the variable named `dataHandler.submitNewTitle`

This way you will always call this function like `dataHandler.submitNewTitle(...)`, so it is unneccessary to give the initial name `submit_new_title` to it.

TLDR: remove `submit_new_title`